neo/Consensus/ConsensusService.cs

```diff
@@ -116,6 +116,11 @@ private void CheckExpectedView(byte viewNumber)
                 if (message is null || message.NewViewNumber < viewNumber)
                     localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeChangeView(viewNumber) });
                 InitializeConsensus(viewNumber);
+                // Save View Change without Transactions for restoring after a restart.
+                var savedTransactions = context.Transactions;
+                context.Transactions = null;
+                context.Save();
+                context.Transactions = savedTransactions;
             }
```

Comment:
What's the reason for saving the context without transactions?

Reply:
Automatically jump the node to the last view it used to know.

Recover can not move nodes backward. Thus, if a node starts on view 0 is becomes prone to receive any Recover payload.
Thus, this change improves the chance of being recovered to a lower view.

Reply:
It is much faster to save the context without the transactions than with them. We only really want to save the view number in this case and want to add minimal performance overhead.